languages/csharp/exercises/concept/operator-overloading/.docs/introduction.md

```diff
@@ -0,0 +1,56 @@
+The principal arithmetic and comparison operators can be adapted for use by your own classes and structs. This is known as _operator overloading_.
+
+Most operators have the form:
+
+```csharp
+static <return type> operaator <operator symbols>(<parameters>);
+```
+
+Cast operators have the form:
+
+```csharp
+static (explicit|implicit) operator <cast-to-type>(<cast-from-type> <parameter name>);
+```
```

Comment:
See my comment on the main (conversation) thread.

Reply:
Addressed.